contracts/misc/Migrations.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.23;
+pragma solidity ^0.5.17;
```

Comment:
If we are using a fixed version in the other contracts, can it be fixed in migrations?